tests/backend/test_generics.py

```diff
@@ -69,27 +68,31 @@ def test_changefeed_class(changefeed_class_func_name, args_qty):
         changefeed_class_func(None, *range(args_qty))
 
 
-@mark.parametrize('db,conn_cls', (
-    ('mongodb', 'MongoDBConnection'),
-    ('rethinkdb', 'RethinkDBConnection'),
-))
 @patch('bigchaindb.backend.schema.create_indexes',
        autospec=True, return_value=None)
 @patch('bigchaindb.backend.schema.create_tables',
        autospec=True, return_value=None)
 @patch('bigchaindb.backend.schema.create_database',
        autospec=True, return_value=None)
 def test_init_database(mock_create_database, mock_create_tables,
```

Comment:
Mmm, in some ways I wonder if this test would be better served by being split off into their own separate tests in `tests/backend/mongo/test_schema.py` and `tests/backend/rethink/test_schema.py`.

The fact that it requires an instance of the connection doesn't make it a *generic* test, and the duplication wouldn't be much different that what we have here already.